src/playbooks/deploy/deploy.yaml

```diff
@@ -12,6 +12,7 @@
     certificates_hostnames:
       - "{{ ansible_fqdn }}"
       - localhost
+    certificates_ca_directory: /root/certificates
```

Comment:
this can also be set in src/vars/default_certificates.yml, which is probably nicer?